core/storage/base_model/gae_models.py

```diff
@@ -1214,6 +1221,13 @@ class BaseSnapshotContentModel(BaseModel):
     # The snapshot content, as a JSON blob.
     content = ndb.JsonProperty(indexed=False)
 
+    @staticmethod
+    def get_deletion_policy():
+        """Collection rights are deleted only if the corresponding collection
```

Comment:
Why this docstring for a "base" model?

Also, not sure how this translates to "not applicable".